blog/kurzarbeit_wenn_der_pruefer_kommt/index.md

```diff
@@ -133,13 +136,9 @@ Arbeitszeitdokumentation und schützt Sie so im Prüfungsfall.
 
 Buchstäblich Hopfen und Malz dürfte bei jenen verloren sein, die Daniel
 Terzenbach als einen der typischen Fälle bezeichnet:
-*„Jemand stellt noch schnell Familienangehörige ein, und das auch noch zu unüblichen und unglaublich hohen Löhnen, um dann für sie Kurzarbeitergeld zu kassieren“.*
-Hier droht, wie Terzenbach sagt, *„…bei einem echten Betrug eine
-Freiheitsstrafe…“.*
+<cite>„Jemand stellt noch schnell Familienangehörige ein, und das auch noch zu unüblichen und unglaublich hohen Löhnen, um dann für sie Kurzarbeitergeld zu kassieren.“</cite>
+Hier droht, wie Terzenbach sagt, <cite>„…bei einem echten Betrug eine
+Freiheitsstrafe…“.</cite>
 
-Bis bald!
-Ihr Pentacode Team
-
-
-(Alle Zitate von Daniel Terzenbach sind einem Interview von „Die Zeit“, Ausgabe
-22/2020 vom 20.Mai 2020 entnommen)
\ No newline at end of file
+[^1]: Alle Zitate von Daniel Terzenbach sind einem Interview von „Die Zeit“, Ausgabe
+22/2020 vom 20.Mai 2020 entnommen
```

Comment:
Fußnoten kannst du in Markdown wie folgt definieren.

Im text: `Ein Zitat bla blah [^1]`
Am ende des Artikels: `[^1]: Quelle oder Erläuterung`